app/services/weets.js

```diff
@@ -0,0 +1,10 @@
+const { Weet } = require('../models');
+const { info } = require('../logger');
+const { databaseError } = require('../errors');
+
+exports.createWeet = (content, user) => {
+  info('Calling weets.createWeet');
```

Comment:
Maybe a log more like: `Creating weet`

Reply:
Nice. I was naming logs like `Calling <service>.<method>` to make them more searchable 😄